code/socialDistribution/views.py

```diff
@@ -437,26 +434,38 @@ def editPost(request, id):
 # https://www.youtube.com/watch?v=VoWw1Y5qqt8 - Abhishek Verma
 
 
-def likePost(request, id):
+def like_post(request, id, post_host):
     """
         Like a specific post
     """
-    post = get_object_or_404(LocalPost, id=id)
+    if post_host == 'remote':
+        post = get_object_or_404(InboxPost, id=id)
+        request_url = post.author.strip('/') + '/inbox'
+        obj = post.public_id.strip('/')
+    else:
+        post = get_object_or_404(LocalPost, id=id)
+        host = request.gethost()
+        request_url = f'http://{host}/{API_PREFIX}/author/{post.author.id}/inbox'
+        obj = f'http://{host}/{API_PREFIX}/author/{post.author.id}/posts/{id}'
+
```

Comment:
Is it possible to make this work by only interacting with InboxPost? Might involve changes to how we display posts on the UI. It would be good if we could treat them the same regardless of whether it is remote or local. I'm just thinking having the id param refer to different object types might get confusing or introduce some bugs.

Reply:
I was trying to achieve that but it seems that it would require a bit more refactoring than I expected. I will make an issue for that and try to address it as soon as I can. How does that sound?

I apologize for the delay in replying to your comments🙇‍♂️ I'll try to be quicker going forward

Reply:
Yes, I think it would. I'm pretty sure it's related to the issue just created by Tyler #176 so that would be a pretty decent issue to tackle on its own. I'm almost certain that tackling that will fix nearly all these other post issues. 

Also, this comment was related to the other comment I left, so I'm fine with it being in another PR. Sorry if I didn't make that clear. 

Please don't apologize, I get that everyone is busy and you reply fast enough as is. I was only wondering about the merge since the PR is approved but you hadn't merged yet.

Reply:
Oh I see, that makes sense. I will try to get a PR up for that over the weekend. We might probably need some test cases as well for this issue. I'll keep you guys posted in the discord and/or on the issue page.

Thanks a lot for explaining and for your understanding! ♥

Reply:
Sounds great. Please assign yourself to #176 or create another issue and assign yourself if you are able to work on it. I may also have time to take a look at it this weekend (depending on how long my other tasks take so not sure) but I won't if you've already got it covered.